src/interactive.rs

```diff
@@ -285,7 +285,7 @@ async fn execute_query1(options: &Options, mut state: &mut repl::State,
             compilation_flags:
                 CompilationFlags::INJECT_OUTPUT_OBJECT_IDS |
                 CompilationFlags::INJECT_OUTPUT_TYPE_NAMES,
-            implicit_limit: state.implicit_limit.map(|x| x as u64),
+            implicit_limit: state.implicit_limit.map(|x| (x+1) as u64),
```

Comment:
Why is this change? It'll break limit set to zero (no limit)

Reply:
"No limit" is represented as `None` both in `state.implicit_limit` and `Execute1::implicit_limit`. And `map` is only applied to `Some(x)`. We avoid using special values like zero use enums instead in Rust.

We need `+1` to display `...` that was broken in protocol 1.0 until this change.